spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -30,15 +30,15 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
-[[microprofile2.0]]
-=== MicroProfile 2.0 (2Q2018)
+[[microprofile2.0.1]]
+=== MicroProfile 2.0.1 (3Q2018)
 
-MicroProfile 2.0 is the sixth release for the https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
```

Comment:
This should now say seventh.

Reply:
I waffled on whether to call 2.0.1 a seventh release or not...  Since it is a micro release, I didn't know if we should update it or not...  Maybe I'll just re-word it to get away from numbering altogether...